try.html

```diff
@@ -553,6 +553,10 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
   <td><img src='/dub/l/vibe-d.svg' alt=''/></td>
   <td><code>https://img.shields.io/dub/l/vibe-d.svg</code></td>
   </tr>
+  <tr><th data-keywords='docker stars'> Docker Stars: </th>
+  <td><img src='/docker/stars/mashape/mockbin.svg' alt=''/></td>
```

Comment:
Would it be feasible to use a different example? This one yields `403 Forbidden`